docs/Governance.md

```diff
@@ -116,15 +116,34 @@ Responsibilities:
 * Manage github settings such as branch protection rules and repository secrets.
 * Resolve any disputes among maintainers.
 
+## Project Milestones
+
+eBPF for Windows uses monthly milestones in the format `YYMM`. So, e.g. the milestone for June, 2024 will be called 2406.
+At any time there are four active milestones in the project. There will be an milestone for the current calendar month (`N`),
+and two others for the next two months (`N+1`), (`N+2`). And there is always a `Backlog` milestone for backlog issues to be
+addressed in the future. In the first triage meeting of a month, the previous milestone must be closed and a new `N+2` milestone must be created.
+
+During the weekly triage meetings, triage drivers MUST set an issue to one of the four milestones according to the priority of the issues.
+
 ## Prioritizing Issues
 
-In the weekly triage meetings, new issues are assigned to collaborators and are set to a monthly milestone by which the issue is expected to be completed. It may not be possible for the assignees to address all the issues assigned to them by their target milestone. As a result issues may pile up in a given milestone. In such cases project admins can request that triage drivers and maintainers use *special labels* to prioritize issues as follows:
+To help prioritize issues, the triage drivers and maintainers MUST use *special labels* to prioritize issues as follows:
 ### P1:
-This label represents highest priority. This label should be used for bugs impacting users, new feature or enhancements that are urgently requested by users. Collaborators  should try to resolve these issues in the **current milestone**.
+This label represents highest priority. This label should be used for bugs impacting users, new feature or enhancements that are urgently
+requested by users. `P1` issues MUST be set to the *current milestone (`N`)*. If there is not enough time left in the current milestone to address
+the `P1` issue, it must be set to the *next milestone (`N+1`)*.
 ### P2:
-This label is used for the next level of priority. Issues marked as P2 for the current milestone may be moved to the *next milestone* after discussing in the weekly triage meeting.
+This label is used for the next level of priority. Issues marked as `P2` can be set to milestones `N`, `N+1` (preferred) or `N+2`.
 ### P3:
-This label is used for low priority issues. Issues marked with this label may be moved to the *next milestone* or the *backlog* milestone, without needing further discussion in the weekly triage meeting.
+This label is used for low priority issues. Issues marked with `P3` label can be set to the milestones `N+1`, `N+2` or `Backlog`.
+`P3` issues must never be set to the current milestone (`N`).
```

Comment:
I'm not sure I agree with this.  I think the project should be encouraging more contributors and allow first time contributors to take a "good first issue" and commit to it in the current milestone even if it's a P3.  I think the rule as stated will dissuade new contributors which I don't think would be healthy for the project.

Reply:
I'd suggest "Issues marked with the `P3` label can be set to any milestone."